ci-operator/step-registry/aws-load-balancer/install-proxy/aws-load-balancer-install-proxy-workflow.metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+	"path": "aws-load-balancer/install-proxy/aws-load-balancer-install-proxy-workflow.yaml",
+	"owners": {
+		"approvers": [
+			"Miciah",
+			"alebedev87",
+			"candita",
+			"miheer",
+			"rfredette",
+			"thejasn"
```

Comment:
No frobware or gcs278?  Here and in the tag-vpc folder.

Reply:
Missed this, added both.